ksql-engine/src/test/java/io/confluent/ksql/function/udf/geo/GeoDistanceTest.java

```diff
@@ -0,0 +1,45 @@
+package io.confluent.ksql.function.udf.geo;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import io.confluent.ksql.function.KsqlFunctionException;
+import static org.junit.Assert.assertEquals;
+
+public class GeoDistanceTest {
```

Comment:
We should also validate that the KUDF fails when less params than 4 or more params than 5 are given to it.

Reply:
new test added for this. seems like a candidate for a more generic test which does this same thing for all registered UDFs ? the FunctionRegistry already has all the info that would be required to do this...